app/controllers/android/httprequest.js

```diff
@@ -0,0 +1,31 @@
+var VolleyPackage = require("com.android.volley.*");
+var JSONPackage = require("org.json.*");
+var activity = new Activity(Ti.Android.currentActivity);
+
+function startRequest() {
+
+    var queue = VolleyPackage.toolbox.Volley.newRequestQueue(activity);
+    var url ="https://appcelerator.com";
+
+    var request = new VolleyPackage.toolbox.JsonObjectRequest(VolleyPackage.Request.Method.GET, url, 
```

Comment:
Also just to note, that the implementation as-is (accessing types in a wildcard package, but not in sub-packages) is consistent with Java coding.